### PR TITLE
Implement basic decoding

### PR DESCRIPTION
**Pull request by foriequal0**
You can preview it here: https://foriequal0.github.io/www-rlp/
Resolves: #2
Resolves: #3 
Resolves: #5

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -5,28 +5,51 @@ import * as RLP from "rlp";
...
+
+    const stock = "0xc701c502c303c104";
     this.state = {
       mode: "encoded",
-      text: "0xc701c502c303c104",
-      rlp: [],
+      text: stock,
+      rlp: {
+        valid: true,
+        decoded: RLP.decode(stock),
```
Comment (sgkim126) [inline src/App.tsx]: Why do you decode it on every change?
Why do not you show an error message when you click the decode button?

File: src/App.tsx [hunk 2]
```diff
@@ -5,28 +5,51 @@ import * as RLP from "rlp";
...
+    valid: boolean;
+    decoded: any;
+  };
 };
 
 class App extends React.Component<{}, AppState> {
   public constructor() {
     super({});
+
+    const stock = "0xc701c502c303c104";
```
Comment (sgkim126) [inline src/App.tsx]: What is it?

File: src/App.tsx [hunk 3]
```diff
@@ -5,28 +5,51 @@ import * as RLP from "rlp";
...
   }
 
   private handleTextChange = (e: any) => {
-    this.setState({ mode: "encoded", text: e.target.value });
+    const { value }: { value: string } = e.target;
+    let rlp;
+    try {
+      rlp = {
+        valid: true,
+        decoded: RLP.decode(value.trim()),
```
Comment (sgkim126) [inline src/App.tsx]: How about adding "0x" prefix automatically?
It seems there is no way to enter the value except a hex string.

Comment (sgkim126) [reply src/App.tsx]: I'll create an issue for it.
Let's discuss it on there.

File: src/App.tsx (hunk 1 above)
Comment (foriequal0) [reply src/App.tsx]: RLP decoding is not expected to be a heavy task.
I wanted to give fast feedback to the user before clicking the button.

File: src/App.tsx (hunk 2 above)
Comment (foriequal0) [reply src/App.tsx]: I want to put there a placeholder. It is a RLP encoded value of `[1,[2,[3,[4]]]]`.

Comment (sgkim126) [reply src/App.tsx]: How about using `placeholder` attribute if you want a placeholder?

**Pull request merged**